packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -199,23 +202,22 @@ export class AppManagementClient implements DeveloperPlatformClient {
     return (await this.session()).token
   }
 
+  async businessPlatformToken(): Promise<string> {
+    return (await this.session()).businessPlatformToken
+  }
+
   async refreshToken(): Promise<string> {
-    const {token} = await ensureAuthenticatedAppManagement([], process.env, {noPrompt: true})
+    const {appToken, businessPlatformToken} = await ensureAuthenticatedAppManagementAndBusinessPlatform(
```

Comment:
appManagementToken I think. appToken implies something to do with this app

Reply:
same above too

Reply:
Updated to `appManagementToken`